images/ceph/Makefile

```diff
@@ -16,7 +16,7 @@ include ../image.mk
 
 # ====================================================================================
 # Image Build Options
-CEPH_VERSION ?= v19.2.1-20250202
+CEPH_VERSION ?= v19.2.2-20250409
```

Comment:
This is also good to backport to 1.16?

Reply:
My thought was to revert the change to the makefile in the 1.16 backport PR since it will result in a conflict.